Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
     "pallets/ajuna-nft-staking/benchmarking",
     "primitives",
 ]
+resolver = "2"
```

Comment:
It has to be placed here unfortunately, since it is part of the `workspace` section, see here -> https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions

Reply:
I meant I prefer it to be on top of the long list of the `members` entry.

Reply:
Ah, true, I will move it then!